i3lock.1

```diff
@@ -444,6 +444,20 @@ Sets the number of minibars to draw on each screen.
 .B \-\-bar\-total\-width
 The total width of the bar. Can be an expression.
 
+.TP
+.B \-\-polygon\-sides
+Draw the indicator as a regular polygon instead of a circle.
```

Comment:
Should document the required arg (number of sides, in addition to current info here)